src/control/lib/ipmctl/nvm_stubs.go

```diff
@@ -1,5 +1,5 @@
 //
-// (C) Copyright 2018-2022 Intel Corporation.
+// (C) Copyright 2022 Intel Corporation.
```

Comment:
The old copyright range is still correct

Reply:
as per previous comment